### ISSUE TITLE
Compiler error CS0433 caused by duplicate types: IdentityServiceCollectionExtensions

### ISSUE THREAD

**Issue by natemcmaster** [aspnet/Identity#1951]
We have two types in the different assemblies with the same name. This can lead to unresolvable compiler errors when referencing Microsoft.Extensions.DependencyInjection.IdentityServiceCollectionExtensions

> error CS0433: The type 'IdentityServiceCollectionExtensions' exists in both 'Microsoft.AspNetCore.Identity, Version=2.1.2.0, Culture=neutral, PublicKeyToken=adb9793829ddae60' and 'Microsoft.Extensions.Identity.Core, Version=2.1.2.0, Culture=neutral, PublicKeyToken=adb9793829ddae60'

https://github.com/aspnet/Identity/blob/release/2.1/src/Identity/IdentityServiceCollectionExtensions.cs
https://github.com/aspnet/Identity/blob/release/2.1/src/Core/IdentityServiceCollectionExtensions.cs

**Comment by ajcvickers** [comment]
@HaoK Which releases were each of these added?

@natemcmaster Is this blocking anything for 2.1/2.2 releases?

**Comment by natemcmaster** [comment]
It looks like the one in Microsoft.AspNetCore.Identity has been around since at least 1.1. The duplicate was added to Microsoft.Extensions.Identity.Core in 2.0 in https://github.com/aspnet/Identity/commit/f555a26b4a554f73eea70b4b34fca823fab9a643

**Comment by natemcmaster** [comment]
No, not blocking release. I just noticed this while trying to reproduce an unrelated error.

**Comment by ajcvickers** [comment]
Thanks Nate. It is a breaking change (binary) to change these now, but we should consider it for 3.0.
